packages/addons/addon-depends/tinc-depends/miniupnpc/package.mk

```diff
@@ -0,0 +1,29 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2018-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="miniupnpc"
+PKG_VERSION="2.0.20180222"
+PKG_SHA256="587944686469d09f739744b3aed70c7ce753a79c40d6f1227f68a3e962665b75"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="http://miniupnp.free.fr"
+PKG_URL="http://miniupnp.free.fr/files/$PKG_NAME-$PKG_VERSION.tar.gz"
```

Comment:
maybe better use https://github.com/miniupnp/miniupnp as they doesn't have any proper releases anyway

Reply:
I tried, but github ships both client and daemon in subdirectories. No idea how to handle that with the LE build system.

Reply:
untested but should work this way, tbh not really a benefit in doing so
```
unpack() {
  mkdir -p $BUILD
    tar -C $BUILD/ --strip-components=1 -xzvf $SOURCES/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.gz miniupnpc/
}
```

**maybe** (not tested at all) `PKG_SOURCE_DIR="${PKG_NAME}-${PKG_VERSION}/miniupnpc"`

Reply:
Changing this does not make much sense at the moment, no?